bind-all.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * @module 101/bind-all
+ */
+
+var isFunction = require('./is-function');
+var keysIn = require('./keys-in');
+/**
+ * Bind a passed object methods.
+ * If methods name to bing are not specified, all the object methods are binded
+ * @function module:101/bind-all
+ *
+ * @param {object} object - object to bind
+ * @param {array|string} [methods] - array or space-separated string containing the names of the methods to bind
+ * @return {object} the binded object
+ */
+module.exports = bindAll;
+
+function bindAll (object, methods) {
+  var keys;
+  if (methods) {
+    // if we passed an array, set keys to the passed array
+    if (Array.isArray(methods)) {
+      keys = methods;
+    } else if (typeof methods === 'string') {
+      // if a string was paased, split it by spaces
+      keys = methods.split(/,\s|[,\s]/);
```

Comment:
@tjmehta I thought of 101 more as a modular alternative to Underscore and LoDash, where all the modules are lighter because they rely on ES5+. I find it especially useful when used in conjunction with Browserify,  so that you load just what you need.

An extra row of code seems to me a reasonable expense to maintain the and Underscore-compatible API. I find it useful just to save keystrokes, typing `'something else'` is much faster than typing `['something', 'else']`, no big deal really.

What should I do about it? Do we keep it?